rosdep/python.yaml

```diff
@@ -4097,31 +4097,16 @@ python-serial:
   gentoo: [dev-python/pyserial]
   openembedded: [python-pyserial@meta-python]
   ubuntu:
-    artful: [python-serial]
+    '*': [python-serial]
     artful_python3: [python3-serial]
-    bionic: [python-serial]
     bionic_python3: [python3-serial]
-    lucid: [python-serial]
-    maverick: [python-serial]
-    natty: [python-serial]
-    oneiric: [python-serial]
-    precise: [python-serial]
-    quantal: [python-serial]
-    raring: [python-serial]
-    saucy: [python-serial]
-    trusty: [python-serial]
+    focal_python3: [python3-serial]
```

Comment:
But we should not add this.  The `_python3` sub-keys were a failed experiment, and they should not be used or added to anymore.  Instead, you should use the `python3-serial` key, which already exists.